### PR TITLE
Add vscode workspace

### PR DESCRIPTION
**Pull request by vladimirlogachev**
I've included a single `haskell.haskell` extension for haskell because it already provides code formatting.

### CODE REVIEW

File: workspace.code-workspace [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "settins": {
+    "[elm]": {
+      "editor.formatOnSave": true
+    },
+    "[haskell]": {
+      "editor.formatOnSave": true
+    }
+  },
+  "extensions": {
+    "recommendations": ["haskell.haskell", "elmtooling.elm-ls-vscode"]
```
Comment (kutyel) [inline workspace.code-workspace]: Can you add ormolu here too?

**Comment by vladimirlogachev** [comment]
The haskell.haskell extension uses default ormolu configuration. It will be good to explicitly provide our configuration in workspace settings, as long as on CI it differs from default one.

File: backend/src/Server.hs [hunk 2]
```diff
@@ -44,9 +44,9 @@ main = do
           hm $
             graphQLApp
               (prometheus p $ libraryServer conn)
-              (Proxy @('Just "Query"))
-              (Proxy @('Just "Mutation"))
-              (Proxy @('Just "Subscription"))
+              (Proxy @( 'Just "Query"))
+              (Proxy @( 'Just "Mutation"))
+              (Proxy @( 'Just "Subscription"))
```
Comment (kutyel) [inline backend/src/Server.hs]: I'm afraid the latest version of ormolu requires you to leave it as before 😉

**Review by kutyel — CHANGES_REQUESTED** [review]
@serras what do you think about this workspaces approach to vscode? 🤔

File: backend/src/Schema.hs [hunk 3]
```diff
@@ -1,4 +1,3 @@
-{-# LANGUAGE CPP #-}
```
Comment (kutyel) [inline backend/src/Schema.hs:1]: 🥳

File: backend/src/Server.hs (hunk 2 above)
Comment (vladimirlogachev) [reply backend/src/Server.hs]: I don't have globally installed ormolu

File: workspace.code-workspace (hunk 1 above)
Comment (vladimirlogachev) [reply workspace.code-workspace]: haskell.haskell already supports formatting via ormolu. Why should another extension be added?

File: backend/src/Server.hs (hunk 2 above)
Comment (kutyel) [reply backend/src/Server.hs]: `nix-env -i ormolu` 😉

Comment (vladimirlogachev) [reply backend/src/Server.hs]: We need an ormolu config, not a "latest version".

File: workspace.code-workspace (hunk 1 above)
Comment (kutyel) [reply workspace.code-workspace]: `sjurmillidahl.ormolu-vscode`

Comment (kutyel) [inline workspace.code-workspace]:
```suggestion
    "recommendations": ["haskell.haskell", "sjurmillidahl.ormolu-vscode", "elmtooling.elm-ls-vscode"]
```

**Review by kutyel — APPROVED** [review]
Awesome, it works like a charm! 🥳

**Pull request merged**